packages/product-react/src/Tabs/Tabs.scss

```diff
@@ -0,0 +1,66 @@
+$easing: cubic-bezier(0.694, 0.0482, 0.335, 1);
+$ray-color-black: black;
+$ray-color-blue-50: blue;
+$hover-border-size: 2px;
+$hover-animation-time: 0.33s;
+
+@mixin tab-border($border-size, $color) {
+  content: '';
+  position: absolute;
+  top: 100%;
+  height: $border-size;
+  left: 0;
+  width: 100%;
+  background-color: $color;
```

Comment:
Nit: to alphabetize

Reply:
If we want to enforce this, should we add stylelint rules?

Reply:
I don't feel strongly about it, make a gh issue if you'd like